deploy/ansible/roles/iota-core-node/templates/docker-compose-iota-core.yml.j2

```diff
@@ -9,58 +9,120 @@
 version: '3.3'
 
 services:
+
+###################
+# IOTA-CORE Nodes #
+###################
+
   iota_core:
     image: {{iota_core_docker_image_repo}}:{{iota_core_docker_image_tag}}
     container_name: iota-core
+    stop_grace_period: 1m
+    restart: unless-stopped
+    ulimits:
+      nofile:
+        soft: 16384
+        hard: 16384
+    ports:
+      - "14666:14666/tcp" # P2P
+      - "6081:6061/tcp"   # pprof
```

Comment:
Why remapping the ports? There is only one instance of a node in every machine

Reply:
Copy paste error :D